modules/backend/formwidgets/RecordFinder.php

```diff
@@ -241,6 +242,10 @@ protected function loadAssets()
      */
     public function getSaveValue($value)
     {
+        if ($this->formField->disabled || $this->formField->hidden) {
+            return FormField::NO_SAVE_DATA;
+        }
```

Comment:
Why doesn't the Form widget handle this for all custom widgets already? Is there an example of other widgets using this pattern already?

Reply:
I'm not sure why the **Form** widget doesn't handle this at its level or even if it should, I mean that we may should keep this logic at the Widget level (and not the Form) to ease the build of customized FormWidget.

Some FormWidget use it (list non-exhaustive below), some don't like [the ColorPicker doesn't](https://github.com/wintercms/winter/blob/develop/modules/backend/formwidgets/ColorPicker.php#L145-L148). 

And from what I discovered, this issue **may** appear with some other FormWidget that I'll test like the ColorPicker (returning null as default), the DataTable ([which return `[]` as default](https://github.com/wintercms/winter/blob/develop/modules/backend/formwidgets/DataTable.php#L116) and may re-init the database json field), the Sensitive and some others.

For the examples of FormWidget that use it:
[DatePicker](https://github.com/wintercms/winter/blob/develop/modules/backend/formwidgets/DatePicker.php#L151-L153)
[MediaFinder](https://github.com/wintercms/winter/blob/develop/modules/backend/formwidgets/MediaFinder.php#L111-L113)
[Relation](https://github.com/wintercms/winter/blob/develop/modules/backend/formwidgets/Relation.php#L174-L176)

Reply:
@RomainMazB are you still interested in writing some tests for this and getting it pushed through?